node/task.ts

```diff
@@ -50,6 +53,8 @@ export enum Platform {
     Linux
 }
 
+const _variableStore: KeyValueStoreInterface = dependencies.variableStore;
```

Comment:
I don't think this is quite right -- clearly this module has a bunch of depenencies that don't come through `dependencies`.

You can use the same mockery pattern here as with everything else.

```
// variablestore.ts
export variableStore = process;

// task.ts
import { variableStore } from './variablestore';

// reference `variableStore` in the code
```